### DISCUSSION TITLE
USB_MSC_EXAMPLE_ERROR

### DISCUSSION THREAD

**Discussion by TigoTas** [espressif/arduino-esp32#7804] [Q&A]
Hi,

I am really not experienced in these topics, so forgive me if my issue prooves to be idiotic.

I am trying to run the "USB_MSC" out of the box, but the code doesn't compile.

I get a "''USBCDC' does not name a type" error message.

Similarly, if I try to run the MSC example from Adafruit's TinyUSB library , also as is, I get a "does not name a type" error message.

My intention is to have the SD card to show as mass storage device on windows, like what LadyAda did on this video.

https://www.youtube.com/watch?v=0bWba0PU4-g

Any help would be appreciated. Perhaps a step-by-step of someone that has had previous experience replicating this functionalty.

Thanks in advance,
[Adafruit_msc_external_flash_sdcard_error.txt](https://github.com/espressif/arduino-esp32/files/10663540/Adafruit_msc_external_flash_sdcard_error.txt)

**Comment by VojtechBartoska** [comment]
@PilnyTomas Can you please help answering this? Thanks!
